models/SearchBuilder.cfc

```diff
@@ -1127,6 +1151,10 @@ component accessors="true" {
 			dsl[ "script" ] = variables.script;
 		}
 
+		if ( !isNull( variables.body ) && !structIsEmpty( variables.body ) ){
```

Comment:
`variables.body` is never going to be null because you are setting it to a struct on init.   You can either just test for emptiness or do the structAppend without the test.  If the `body` struct is empty, the structAppend does nothing.